pymeasure/instruments/keithley/keithley2182.py

```diff
@@ -62,61 +62,58 @@ def __init__(self, instrument, id_):
 
     voltage_range = Channel.control(
         ":SENS:VOLT:CHAN{ch}:RANG?", ":SENS:VOLT:CHAN{ch}:RANG %g",
-        """ A floating point property that controls the positive full-scale
-        measurement voltage range in Volts, which can take values from 0 to
-        120 V (Ch. 1) or 12 V (Ch. 2). Auto-range is automatically disabled
-        when this property is set. """,
+        """Control the positive full-scale measurement voltage range in Volts.
+        Valid ranges are from 0 to 120 V for Ch. 1, and 0 to 12 V for Ch. 2.
+        Auto-range is automatically disabled when this property is set.""",
         validator=truncated_range,
         values=[0, 120],
         dynamic=True,
     )
     voltage_range_auto = Channel.control(
         ":SENS:VOLT:CHAN{ch}:RANG:AUTO?", ":SENS:VOLT:CHAN{ch}:RANG:AUTO %d",
-        """ A boolean property that controls whether auto voltage ranging is
-        enabled. Valid values are True and False.""",
+        """Control whether auto voltage ranging is enabled.
+        Valid values are True and False.""",
         validator=strict_discrete_set,
         values={True: 1, False: 0},
         map_values=True,
     )
```

Comment:
For boolean properties, it is good to use a name like `voltage_range_auto_enabled`, which makes the code descriptive:
`voltage_range_auto_enabled = True` is clear, while otherwise you never know, what True means: enabled or disabled.